CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ if(LSTM_TORCH_LIB_ACTIVE)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${TORCH_CXX_FLAGS}")
 endif()
 
-find_package(Boost 1.72.0 REQUIRED)
+find_package(Boost 1.72.0 REQUIRED COMPONENTS filesystem)
```

Comment:
Just noticed this come through and I wanted to weigh in on the inclusion of `boost:: filesystem`. So far we have been careful to use header only boost components, and the discussion of `boost:: filesystem` in the past led us to steer clear of it to keep the binary dependency out of the equation. We can revisit this topic, but I wanted to get this warning up before we got to invested.